make/common/Modules.gmk

```diff
@@ -64,11 +64,9 @@ ifeq ($(INCLUDE_JVMCI), false)
   MODULES_FILTER += jdk.internal.vm.ci
 endif
 
-# Filter out Graal specific modules if Graal is disabled
-ifeq ($(INCLUDE_GRAAL), false)
-  MODULES_FILTER += jdk.internal.vm.compiler
-  MODULES_FILTER += jdk.internal.vm.compiler.management
-endif
+# Filter out Graal specific modules
+MODULES_FILTER += jdk.internal.vm.compiler
```

Comment:
If we are unconditionally filtering out these modules, then why leave the module-info.java files in at all?

Reply:
We filter out because we can't build Graal anymore. But we need these module-info.java files because JVMCI's module-info.java references them:
https://github.com/openjdk/jdk/blob/master/src/jdk.internal.vm.ci/share/classes/module-info.java#L26

Otherwise we can't build JVMCI which we continue to support.

I filed followup RFE to implement Alan's suggestion to use Module  API which will allow to remove these files later:
https://bugs.openjdk.java.net/browse/JDK-8265091

Reply:
Right, I thought I saw something about modules that Alan commented on, but couldn't find it. All good then.